gapic/src/main/com/google/gapid/views/FramebufferView.java

```diff
@@ -106,6 +121,20 @@ public FramebufferView(Composite parent, Models models, Widgets widgets) {
     // Work around for https://bugs.eclipse.org/bugs/show_bug.cgi?id=517480
     Widgets.createSeparator(toolBar);
 
+    RENDER_SHADED = Paths.renderSettings(MAX_SIZE, MAX_SIZE, Path.DrawMode.NORMAL, 
+      models.settings.preferences().getDisableReplayOptimization());
+
+    RENDER_OVERLAY = Paths.renderSettings(MAX_SIZE, MAX_SIZE, Path.DrawMode.WIREFRAME_OVERLAY, 
+      models.settings.preferences().getDisableReplayOptimization());
+    
+    RENDER_WIREFRAME = Paths.renderSettings(MAX_SIZE, MAX_SIZE, Path.DrawMode.WIREFRAME_ALL, 
+      models.settings.preferences().getDisableReplayOptimization());
+    
+    RENDER_OVERDRAW = Paths.renderSettings(MAX_SIZE, MAX_SIZE, Path.DrawMode.OVERDRAW, 
+      models.settings.preferences().getDisableReplayOptimization());
```

Comment:
The settings can change, and if they do, future requests should honor the change. We now need to create them each time we send a request.
I'd suggest to create an enum with a `Path.RenderSettings getRenderSettings(Settings settings)` method.

Reply:
Should we consider doing so for UsageHints as well? I am working on the UI strip, and I needed to add a new one with a different size, but also shouldn't I be using UsageHints.preview = true for it?

Reply:
No, the UsageHints are constant. We use preview for thumbnails. The idea behind the usage hints is to allow the server to prioritize replays. If `primary` is true, it's meant to say that the user is blocked on the data, so it should run first. This should be used when requesting the FB contents for the main view, but not for previews/thumbnails. For the attachment selector, you should use preview.